rs-backend/src/api.rs

```diff
@@ -0,0 +1,167 @@
+// Types implement both serialize and deserialize despite conceptually needing
+// only one for testing.
+
+pub mod summary {
+    use std::collections::HashMap;
+
+    /// One decimal place rounded percent
+    type Percent = String;
```

Comment:
I think a display impl isn't enough: we'd have to implement serialize/deserialize ourselves. That's the main reason for doing this; aside from minor concerns related to having to export it (making summaries unique in that aspect of exporting something other than Request/Response).